docs/Assets.md

```diff
@@ -0,0 +1,105 @@
+# Assets and JSON module imports
+
+UI5 Web Components aim to be feature rich and with a minimal code footprint at the same time. In order to achieve this, 
+most UI5 Web Components packages ship their assets as `.json` files while also providing respective public module imports for them.
+
+The assets in question could be i18n texts, icons, additional themes parameters, CLDR, etc...
+
+Currently our npm packages follow the scheme:
+
+`@ui5/<PACKAGE_NAME>/dist/assets/*`
+(for the assets themselves)
+
+`@ui5/<PACKAGE_NAME>/dist/json-imports/*`
+(for the modules that provide the assets)
+
+<a name="packages"></a>
+## Packages
+
+### `base` package
+
+Asset type | Asset path | Public Module Import
+------|------------------|---
+CLDR | `@ui5/webcomponents-base/dist/assets/cldr/*` | `@ui5/webcomponents-base/dist/json-imports/LocaleData.js`
```

Comment:
describing the assets doesn't make much sense, as we haven't documented the registries and the asset paths are not easily configurable. Le's stick to the json imports for now.

Reply:
I put the assets paths here, because people might have their own "assets" directory and they should have a clear idea where are assets are so they can write stricter rules for rollup/webpack, if needed.

Reply:
I was looking at this as a consumption guide, authoring packages needs much more documentation and it belongs there, but i'm fine to leave it here.